### PR TITLE
feat: add label placement, value and percentage to funnel chart

### PR DESCRIPTION
**Pull request by magnew**
Closes: #10488 

### Description:

Add label properties to funnel chart:
- placement
- show value (without hover)
- show percentage (without hover)

Chart with label: right, value and percentage:
Screenshot 2024-06-24 at 19 06 12

Config:
Screenshot 2024-06-24 at 19 06 48

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by hamzahc1** [comment]
Shouldn't the label text always be black/grey? it's weird that it's yellow, blue, red etc - it means it's hard to read depending on the colourscheme you've picked...

File: packages/frontend/src/hooks/echarts/useEchartsFunnelConfig.ts [hunk 1]
```diff
@@ -54,22 +73,39 @@ const useEchartsFunnelConfig = (isInDashboard: boolean) => {
...
+                            maxValue: chartConfig.maxValue,
+                        });
+
+                    return `${marker}<b>${name}</b><br /> Value: ${formattedValue} <br/> Percent: ${percentOfMax}%`;
                 },
             },
             label: {
                 show: true,
-                position: 'inside',
+                position: label?.position || FunnelChartLabelPosition.INSIDE,
```
Comment (joaoviana) [inline packages/frontend/src/hooks/echarts/useEchartsFunnelConfig.ts:88]: we could add here `color: 'black'` if the position is different from `inside`?

**Review by joaoviana — CHANGES_REQUESTED** [review]
I think we can address @hamzahc1's point - left a suggestion on how to add it. 

Great stuff, tested in Render 👍

File: packages/frontend/src/hooks/echarts/useEchartsFunnelConfig.ts (hunk 1 above)
Comment (magnew) [reply packages/frontend/src/hooks/echarts/useEchartsFunnelConfig.ts:88]: Done

**Review by joaoviana — APPROVED** [review]
Checked in render and label is `black` when it's not inside. 
If inside, echarts does a contrast check and applies the correct color

**Pull request merged**